### ISSUE TITLE
Make enable_ssl.sh run without root privilege

### ISSUE THREAD

**Issue by mekya** [ant-media/Ant-Media-Server#4914]
`enable_ssl.sh` requires root privilege to run. We plan to support installing SSL through web panel and `antmedia` user does not have root privileges so that we cannot enable SSL through web panel. 

It's required to run the enable_ssl.sh without root privilege. When I check that it seems that if we use `webroot` plugin for certbot, it may be doable to run the `enable_ssl.sh` without root privilege

**Comment by muratugureminoglu** [comment]
Hi @mekya 

When I tried with Webroot, port 80 must be open. This requires additional requirements.  

The example command is as follows.
```
certbot certonly --webroot -w /usr/local/antmedia -d ssl-test.antmedia.cloud --logs-dir /usr/local/antmedia/log/ --config-dir /usr/local/antmedia/ --work-dir /usr/local/antmedia/
```
Instead of the above method, I did it with a little change on the system side, but I still need time for security hardening.

![Screenshot at 2023-05-08 12-59-25](https://user-images.githubusercontent.com/6337605/236800314-758c39d2-4213-4a91-a30d-eb5bbced580a.png)

And there are some bugs. I will report them too.

![Screenshot at 2023-05-08 13-10-52](https://user-images.githubusercontent.com/6337605/236800408-038156b9-1b9a-45b0-9d69-c90905c3d866.png)

**Comment by rjtoledo** [comment]
Hi @mekya  I'm using version 2.6.1 running in Ubuntu 18.04 I don't see any SSL settings in my dashboard.

**Comment by mekya** [comment]
Hi Guys,
Thank you for raising asking question. I appreciate that @rjtoledo @k99IT.

The web panel side of the implementation has not been merged yet. .If we don't encounter any unexpected scenario, we plan to make it available in 2.6.2 and ETA for 2.6.2 is likely next week. 

PS: We've started to release frequently and I mean every 2-4 weeks.  I hope we'll keep the momentum with your support :)

Cheers
